styles/globals.css

```diff
@@ -46,9 +48,19 @@ footer {
   text-align: center;
   position: relative;
   bottom: 0;
-  padding-top: 20px;
-  margin-bottom: 100px;
+  padding: 20px 0px;
+  background-color: #041484;
+  color: #f4f4f4;
+  font-weight: 100;
+  margin-top: 10px;
+}
+
+@media only screen and (min-width: 900px) {
+  footer {
+    position: absolute;
+  }
```

Comment:
@kachwalUttamsharma Can you please tell me why are we using this?

Reply:
I used it to make responsive, in between let me re-verify it again.